authors/apps/authentication/views.py

```diff
@@ -2,11 +2,16 @@
 from rest_framework.generics import RetrieveUpdateAPIView
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
-
+from rest_framework.views import APIView
+from django.contrib.auth.tokens import default_token_generator
+from django.contrib.sites.shortcuts import get_current_site
+from .models import User
+from authors.apps.core.mailer import SendMail
 from .models import User
+
 from .renderers import UserJSONRenderer
 from .serializers import (
-    LoginSerializer, RegistrationSerializer, UserSerializer
+    LoginSerializer, RegistrationSerializer, UserSerializer, ForgetPasswordSerializer, ResetPasswordSerializer
```

Comment:
@mikenthiwa also with the serializer name use `ForgtoPasswordSerializer` as the name.